controllers/game.controller.js

```diff
@@ -70,3 +71,12 @@ exports.getUsers = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+  deleteComment(comment_id)
+    .then((comment) => {
+      res.status(204).send({ comment });
```

Comment:
express doesn't send back anything from a delete request, so you can just leave it as .send()